lib/options.js

```diff
@@ -210,6 +210,15 @@ module.exports = {
         description: 'Whether to set optional parameters as disabled',
         external: true,
         usage: ['CONVERSION']
+      },
+      {
+        name: 'Keep implicit headers',
+        id: 'keepImplicitHeaders',
+        type: 'boolean',
+        default: false,
+        description: 'Whether to keep implicit headers from the OpenAPI specification,which are removed by default.',
```

Comment:
Forgot to add comment here, can we add empty space before comma.

`specification, which`

Reply:
Corrected the spelling mistake.